docs/general/whats-new.md

```diff
@@ -30,6 +30,10 @@
 
 - The validity of an image's signature can be [verified](../articles/verifying-signatures.md) by zot. Users can upload public keys or certificates to zot.
 
+### LDAP credentials stored apart from configuration
```

Comment:
"apart" -> "separate"

(maybe add this here also)

This model allows for separation of configuration and credentials.
(rephrase pls)

Reply:
Changed:  "LDAP credentials stored separately from configuration"